### ISSUE TITLE
evaluation

### ISSUE THREAD

**Issue by dmpe** [dmpe/CaseSolvingSeminar#13]
http://www.saedsayad.com/model_evaluation_c.htm
http://machinelearningmastery.com/classification-accuracy-is-not-enough-more-performance-measures-you-can-use/
http://blog.tryolabs.com/2013/03/25/why-accuracy-alone-bad-measure-classification-tasks-and-what-we-can-do-about-it/
http://www2.cs.uregina.ca/~dbd/cs831/notes/ROC/ROC.html
http://radimrehurek.com/data_science_python/

**Comment by dmpe** [comment]
http://www2.cs.uregina.ca/~dbd/cs831/notes/ROC/ROC.html
http://gim.unmc.edu/dxtests/roc3.htm
http://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score
http://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html
